spk-config.yaml

```diff
@@ -5,7 +5,26 @@ azure_devops: # Should this live under the `spk` configuration?
   manifest_repository: "dev.azure.com/bhnook/fabrikam/_git/materialized" # Repository URL that is configured for flux. This holds the kubernetes manifests that is generated by fabrikate.
   org: "https://dev.azure.com/bhnook" # Your AzDo Org
   project: "fabrikam" # Your AzDo project
-
+  variable_group:
+    name: "myvg"
+    description: "myvg description"
+    variables: # Either this or Key Value provider should be configured for this variable group
+      var1:
+        value: value1
+        isSecret: true
+      var2:
+        value: value2
+        isSecret: false
```

Comment:
do you think it makes sense to prompt the user to enter values for their variables (or take them via command line arguments), instead of saying var1 and var2, which does not scale in case they want 10 variables in their VG?

Reply:
I agree about prompting command line arguments for one time use only values. I know the infra folks were following this flow. spk-config holds values for things that are needed repeatedly.

Reply:
Prompts for each command also has scalability issues i.e. 10 prompts for 10 variables. I will enable accepting a payload form the file with other required variables.